README.md

```diff
@@ -1,7 +1,133 @@
+<h1 align="center">
+  <img src="media/realmq-platform-logo.svg" alt="RealMQ Logo" />
+  <br><br>
+</h1>
+
+![RealMQ Platform Build Status](https://drone.rmq.ovh/api/badges/RealMQ/realmq-platform/status.svg)
 [![FOSSA Status](https://app.fossa.io/api/projects/git%2Bgithub.com%2Frealmq%2Frealmq-platform.svg?type=shield)](https://app.fossa.io/projects/git%2Bgithub.com%2Frealmq%2Frealmq-platform?ref=badge_shield)
 
+# RealMQ :rocket: Open Source Real-time Messaging Platform
+
+[RealMQ](https://realmq.com) is a free and open source backbone for real-time telecommunication.
+Our core values are **performance** and **integrability**.
+The platform is built around a convenient REST-API to manage access control, data channels and users/devices.
+Messages, events and data are exchanged via the battle proven MQTT protocol which is lightweight, fast and well established in the world of IoT.
+Clients connect to our highly scalable MQTT broker that takes care of delivering messages to all connected subscribers.
+On top of this slim but flexible core you can build awesome realtime products and services around use-cases like:
+
+* Device to device communication
+* Exchange signals / send and receive realtime events
+* Broadcast messages or events to segmented users
+* Aggregate and dispatch data
+* Emit sensor/IoT Device data
+* Chat
+
+## Features
+### REST API
+We follow the API first approach, as we see our API as the central component in our architecture.
+Our mission is to build a developer friendly API with industry standard patterns and
+tools like RESTful design principles and an OpenAPI specification.
+It goes without saying that all requests are TLS encrypted and authentication is backed by security tokens (Bearer authentication).
+
+### Message Persistence
+Message history can be enabled and fine tuned on channel level.
+Once enabled, all messages published to the channel will be kept for a configured amount of time.
+Persisted messages can be retrieved via API history resource.
+
+<details>
+<summary>Code sample</summary>
+
+```js
+// Create a new channel that keeps messages for 2 weeks 
+const channel = await realmq.channels.create({
+  history: '2 weeks'
+});
+
+// Retrieve the history (last 20 messages)
+const history = await realmq.history.query({
+  channel,
+  limit: 20
+});
+
+```
+
+</details>
+
+### Custom Ids
+It’s just a little detail that manifests the seamless integration into your stack:
+All API resources can be operated with optional custom ids.
+Here is an example of how working with custom ids in our nodejs sdk looks like:
+
+<details>
+<summary>Code sample</summary>
+
+```js
+// Create and lookup resources with custom ids
+const channel = await realmq.channels.create({ id: 'custom-channel-id' });
+
+// Or get an id auto generated
+const subscription = await realmq.subscriptions.create();
+subscription.id // => 20f62e87-e689-4a11-bcf3-a78026fffd85
+
+```
+
+</details>
+
+### Multi-Tenancy / Projects
```

Comment:
IMHO the headline should be `Realms` and the text explaining that term with multi tenancy and projects.